src/components/Highlight.vue

```diff
@@ -1,17 +1,23 @@
 <template>
-  <span
-    :class="suit()"
-    v-html="innerHTML"
+  <ais-highlighter
+    :hit="hit"
+    :attribute="attribute"
+    :highlighted-tag-name="highlightedTagName"
+    :suit="suit"
+    highlight-property="_highlightResult"
+    pre-tag="<mark>"
+    post-tag="</mark>"
```

Comment:
We need to be able to fall back to `__ais-highlight__` / `__/ais-highlight__` if escape-highlight is false of the parent hits or infinite hits

Reply:
yeah, I said [that](https://github.com/algolia/vue-instantsearch/pull/783#discussion_r433163166), but I realized that we have only `hit` inside ais-highlight. We don't have `hits`, so cannot tell `hits.__escaped` is true or not.

Reply:
Previously, Vue InstantSearch just set innerHTML with

```js
computed: {
    innerHTML() {
      return instantsearch.highlight({
        attribute: this.attribute,
        hit: this.hit,
        highlightedTagName: this.highlightedTagName,
      });
    },
  },
```

and I see `highlight` function in InstantSearch.js already assumes the hit is escaped.
What do you think?